README.md

```diff
@@ -117,8 +119,32 @@ pod install
 
 The SDK must be initialized with the short alphanumeric
 [public API key](https://help.klaviyo.com/hc/en-us/articles/115005062267#difference-between-public-and-private-api-keys1)
-for your Klaviyo account, also known as your Site ID. Initialization is done in the native layer, and must occur before
-any other SDK methods can be invoked. Follow the native SDK instructions for initialization, and refer to the
+for your Klaviyo account, also known as your Site ID.
+
+Initialization can either be done from your app's React native or native code, and must occur before
+any other SDK methods can be invoked so that Klaviyo SDK can track profiles, events and push tokens
+towards your company.
+
+The decision to initialize the SDK from the React Native or the native code depends on your app's architecture.
+
+Finally, the SDK initialization needs to be done just once either in your react native or native code.
+It's not required to initialize the SDK in both places.
```

Comment:
```suggestion
for your Klaviyo account, also known as your Site ID.

Initialize *must* be called prior to invoking any other SDK methods so that Klaviyo SDK can track profiles, events and push tokens toward the correct Klaviyo account. Any SDK operations invoked before initialize will be dropped, and result in a logged error.

You can call `initialize` from your app's React Native layer or from the platform-specific native code. 
This decision is dependent only upon your app's architecture. It is not required to initialize the SDK in both places!
Note: It is safe to re-initialize, e.g. if your app can connect to more than one Klaviyo account. 
```